med-web-app-front/src/services/dicom-anonymizer.service.js

```diff
@@ -16,6 +16,9 @@ class DicomAnonymizerService {
         let dicomContent = dcmjs.data.DicomMessage.readFile(arrayBuffer)
         let dicomTagsDataset = dcmjs.data.DicomMetaDictionary.naturalizeDataset(dicomContent.dict)
 
+        console.log(dicomTagsDataset.StudyInstanceUID);

```

Comment:
Codacy has a fix for the issue: [Replace `····console.log(dicomTagsDataset.StudyInstanceUID);␍` with `);⏎⏎····console.log(dicomTagsDataset.StudyInstanceUID);`](https://app.codacy.com/gh/danielsheh02/Medical-Web-App/pullRequest?prid=9534713)
```suggestion
    );
```